spec.html

```diff
@@ -45238,14 +45244,19 @@ <h1>
             1. Assert: If we return here, the generator either threw an exception or performed either an implicit or explicit return.
             1. Remove _genContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
             1. Set _generator_.[[GeneratorState]] to ~completed~.
-            1. Once a generator enters the ~completed~ state it never leaves it and its associated execution context is never resumed. Any execution state associated with _generator_ can be discarded at this point.
-            1. If _result_.[[Type]] is ~normal~, let _resultValue_ be *undefined*.
-            1. Else if _result_.[[Type]] is ~return~, let _resultValue_ be _result_.[[Value]].
+            1. Once a generator enters the ~completed~ state it never leaves it and there is never again a transfer of control to its associated execution context. Any execution state associated with _generator_ can be discarded at this point.
```

Comment:
```suggestion
            1. NOTE: Once a generator enters the ~completed~ state it never leaves it and there is never again a transfer of control to its associated execution context. Any execution state associated with _generator_ can be discarded at this point.
```

Reply:
That'd be fine with me, but since it isn't a Note in the status quo, I'll want the other editors to agree.

Reply:
I agree with Michael; the fact that it isn't a NOTE in the status quo bothers me too. Happy to split that into its own PR though.

Reply:
Opened #3026 for this.